### DISCUSSION TITLE
[Form] Pesquisa Web Semântica

### DISCUSSION THREAD

**Discussion by ronysilvati** [frontendbr/forum#892] [Mostre seu trabalho]
Olá galera do front! Estou fazendo um artigo sobre Web Semântica e WebComponents e gostaria da participação de vocês!

Montei um questionário que é respondido rapidinho (5 perguntas somente). Agradeço a galera que puder contribuir!
Segue o link: 

https://goo.gl/forms/9VNV52ZJXPysGclt1

**Comment by renanlara** [comment]
Mandei lá!

**Comment by felipemfp** [comment]
Respondido! :smile:

**Comment by isaacguerreir** [comment]
Respondido também. Avise aqui quando o artigo estiver pronto.

**Comment by ronysilvati** [comment]
Valeu galera! Pode deixar que aviso e posto o artigo também!

**Comment by dionarodrigues** [comment]
Respondido.

**Comment by AlexsandroSA** [comment]
Respondido!

**Comment by vaffonso** [comment]
Feito

**Comment by tarciozemel** [comment]
Feito. No aguardo dos resultados. :+1:

**Comment by ronysilvati** [comment]
Blz! Logo que pronto posto os resultados!

**Comment by TiagoDanin** [comment]
Feito! Seria uma boa tem uma opção na primeira pergunta para "Back-end"

**Comment by ronysilvati** [comment]
Bom dia pessoal. Segue o artigo o qual a pesquisa fez parte. Obrigado a todos.

https://github.com/frontendbr/forum/issues/1112
